src/test/java/cosminsechel/tests/seleniumpart1/Demoqatema.java

```diff
@@ -0,0 +1,29 @@
+package cosminsechel.tests.seleniumpart1;
+
+import cosminsechel.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Demoqatema {
+
+    public static void main(String[] args) throws InterruptedException {
+        BrowserManager.createDriver();
+        WebDriver driver = new ChromeDriver();
```

Comment:
This line should be using the BrowserManager.createDriver() method -> update existing line to ->
WebDriver driver = BrowserManager.createDriver();